samples/copy-from-container/pom.xml

```diff
@@ -0,0 +1,89 @@
+<project xmlns = "http://maven.apache.org/POM/4.0.0"
```

Comment:
Project in "samples" directory use different formatting of this header:

```xml
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <modelVersion>4.0.0</modelVersion>
```

Reply:
It looks like this file doesn't follow formatting of existing POMs from the "samples" directory, where 2 space indentation is used (while this file uses 3 spaces).